mex/common/resources/vocabularies/resource-type-general.json

```diff
@@ -90,5 +90,41 @@
       "de": "Fragebogen",
       "en": "Questionnaire"
     }
+  },
+  {
+    "identifier": "https://mex.rki.de/item/resource-type-general-9",
+    "inScheme": "https://mex.rki.de/item/resource-type-general",
+    "notation": "1.1",
+    "prefLabel": {
+      "de": "Sequenzdaten",
+      "en": "Sequence data"
+    }
+  },
+  {
+    "identifier": "https://mex.rki.de/item/resource-type-general-10",
+    "inScheme": "https://mex.rki.de/item/resource-type-general",
+    "notation": "1.2",
+    "prefLabel": {
+      "de": "Surveillancedaten",
+      "en": "Surveillance data"
+    }
+  },
+  {
+    "identifier": "https://mex.rki.de/item/resource-type-general-10",
```

Comment:
we have 10 twice here

Reply:
Thank you!
Luca has to review this as well. She will take care of it after she's back.